README.md
# practice
practice scripts


history.txt
    3  cd ../..
    4  cd shell_scripting
    5  ls
    6  cat add_number.sh
    7  logout
    8  ps -ef | grep tomcat
    9  sudo -i
   10  ls -lrt
   11  sudo -i
   12  ip a
   13  systemctl restart jenkins
   14  systemctl status jenkins
   15  sudo -i
   16  logout
   17  ifconfig
   18  ip
   19  ip r
   20  sudo -i
   21  sudo -i
   22  ls
   23  systemctl status jenkins
   24  sudo -i
   25  systemctl status jenkins
   26  sudo -i
   27  clear
   28  clear
   29  where is maven
   30  whereis maven
   31  cd /etc/
   32  ls
   33  cd maven
   34  ls
   35  ls -lrt
   36  cd ..
   37  ls
   38  mvn clean install
   39  ls
   40  whereis maven
   41  sudo -i
   42  sudo -i
   43  javac -version
   44  ls
   45  cd repo_git
   46  ls
   47  cd Maven_Examples
   48  ls
   49  cd sample_java
   50  ls
   51  mvn clean install
   52  whereis mvn
   53  ls
   54  ls -lrt
   55  cd dir1
   56  ls
   57  vi deep1
   58  clear
   59  vi deep1
   60  cd dir1
   61  ls
   62  cd dir1
   63  ls
   64  cd dir1
   65  ls
   66  cd dir1
   67  ls
   68  cd ..
   69  mkdir amzon
   70  ls
   71  cp -r amzon dir1
   72  ls
   73  cd dir1
   74  ls
   75  rm -rf amzon
   76  ls
   77  cp deep1 /home/ec2-user/amzon
   78  ls
   79  cd ../amzon
   80  ls
   81  rm -f deep1
   82  ls
   83  cd ../dir1
   84  cat ht
   85  wc -l ht
   86  wc -w h
   87  wc -w ht
   88  wc -c ht
   89  cat deep1
   90  grep "good" deep1
   91  grep -e "good" -e make deep1
   92  grep -w "make" deep1
   93  grep -l "well" deep1
   94  grep -l -R "well" deep1
   95  grep "^ they " deep1
   96  grep "^ they" deep1
   97  grep "^ if" deep1
   98  grep -i "^ if" deep1
   99  ls -lrt
  100  cat deep1
  101  grep "^ Resource" deep1
  102  vi deep1
  103  grep "^ unix" deep1
  104  grep "^unix" deep1
  105  grep "^Resource" deep1
  106  cat deep1
  107  grep "^they" deep1
  108  vi deep1
  109  grep "linux$" deep1
  110  grep -c "linux" deep1
  111  grep -V "linux" deep1
  112  grep -v "linux" deep1
  113  grep -v "unix" deep1
  114  grep -v "essay" deep1
  115  cat deep1
  116  head -2 deep1
  117  head -1
  118  head -1 deep1
  119  head -5 deep1 | tail -1
  120  ls -lrt | wc -l
  121  lcd dir1
  122  cd dir1
  123  ls
  124  car deep1
  125  cat deep1
  126  head -8 deep1
  127  tail -2
  128  tail -2 deep1
  129  tail -5
  130  tail -5 deep1
  131  head deep1
  132  tail deep1
  133  sed 's/good/bad/g' deep1
  134  sed -i 's/good/bad/g' deep1
  135  cat deep1
  136  clear
  137  sed '$d' deep1
  138  cat deep1
  139  clear
  140  cd dir1
  141  ls
  142  sed -n '2P' deep1
  143  sed '2d' deep1
  144  sed '2,5d' deep1
  145  sed '2,6d' deep1
  146  cat deep1
  147  sed -n '$d' deep1
  148  sed '$d' deep1
  149  ls -lrt
  150  ls -lrt | cut -d " " -f
  151  ls -lrt | cut -d " " -f{print $1}' 
  152  cut -d " " -f{print $1}'
  153  awk -F " " -f{print $1}'
  154  awk -F " " -f{print $1}' | ls -lrt
  155  awk -F " " -f{print $1}' |ls -lrt
  156  cd dir1
  157  ls
  158  cat gg
  159  cat 4tab.sh
  160  cat exel
  161  cut -d " " -f 1 exel
  162  cut -d " " -f 3 exel
  163  cut -d " " -f 2 exel
  164  cat exel
  165  cut -d " " -f 1,2 exel
  166  cut -d " " -f 3,4 exel
  167  cut -d " " -f 4,5 exel
  168  awk -F " " '{print $1}' exel
  169  awk -F " " '{print $2}' exel
  170  awk -F " " '{print $(NF)}' exel
  171  awk -F " " '{print $(NF-1)}' exel
  172  awk -F " " '{print $NF}' exel
  173  awk -F " " '{print $(NF-2)}' exel
  174  cat exel
  175  ls -lrt
  176  ls -lrt | cut -d " " -f 1
  177  ls -lrt | cut -d " " -f 2
  178  ls -lrt | cut -d " " -f 3
  179  ls -lrt | cut -d " " -f 4,5
  180  ls -lrt | cut -d " " -f NF
  181  ls -lrt | cut -d " " -f $(NF)
  182  ls -lrt | awk -F " " '{print 1,2}'
  183  ls -lrt
  184  ls -lrt | awk -F " " '{print $1,2}'
  185  clear
  186  ls -lrt
  187  ls -lrt | awk -F " " '{print $(NF-1)}'
  188  ls -lrt | awk -F " " '{print $(NF-1,2)}'
  189  ls -lrt | awk -F " " '{print $(NF-5)}'
  190  ls -lrt | awk -F " " '{print $(NF -5)}'
  191  ls -lrt | awk -F " " '{print $(NF -2)}'
  192  ls -lrt | awk -F " " '{print $(NF -3)}'
  193  ls -lrt | awk -F " " '{print $(NF)}'
  194  ls -lrt | awk -F " " '{print $NF}'
  195  sed '2d' deep1
  196  sed '$d' deep1
  197  sed -n '2,4P' deep1
  198  sed -n '$2P' deep1
  199  sed -n '4$P' deep1
  200  sed -n '4,$P' deep1
  201  cd dir1
  202  find . -type f -perm 0777
  203  find . -type f -perm 0776
  204  find . -type f -perm 0775
  205  find . -type f -perm 0774
  206  find . -type f -perm 774
  207  find . -type f -perm 772
  208  ls
  209  ls -lrt
  210  find . -tpye f ! -empty
  211  find . -type f ! -empty
  212  find . -type -mtime -90
  213  find . -mtime -90
  214  find . -mmin -90
  215  find. -mmin -90
  216  find . -mmin -30
  217  cd dir1
  218  cd dir1
  219  ls
  220  vi $?.sh
  221  ./ $?.sh
  222  ls
  223  sh $?.sh
  224  l

history.sh
    1  ls -lrt
    2  cat check
    3  cd shell_scripting
    4  ls -lrt
    5  cat file_dir.sh
    6  cat add_sub_mul_div.sh
    7  cd shell_scripting
    8  ls -lrt
    9  cd shell_scripting
   10  ls -lrt
   11  vi add_sub_mul_div.sh
   12  sh add_sub_mul_div.sh
   13  vi add_sub_mul_div.sh
   14  vi file_dir.sh
   15  sh file_dir.sh
   16  vi file_dir.sh
   17  vi while.sh
   18  cd shell_scripting
   19  ls -lrt
   20  vi while.sh
   21  pwd
   22  cd linux_command
   23  ls
   24   cd /home/ec2-user/
   25  ls -lrt
   26  rm -rf dir7
   27  rm -rf touch
   28  rm -rf file1.pdf
   29  rm -rf dir4
   30  rm -rf dir9
   31  rm -rf file1
   32  rm -rf dir66
   33  rm -rf B
   34  rm -rf dir34
   35  ls -lrt
   36  cd dir1
   37  ls
   38  cd shell_scripting
   39  cd shell_scripting
   40  cd ..
   41  cd shell_scripting
   42  l;s
   43  ls
   44  ls
   45  rm b
   46  rm -rf b
   47  rm -rf dir101
   48  rm -rf dir2
   49  sudo su
   50  cd shell_scripting
   51  ls -lrt
   52  cp read.sh /home/ec2-user/dir1
   53  ls -lrt
   54  cd ..
   55  ls
   56  pwd
   57  cd dir1
   58  pwd
   59  cd shell_scripting
   60  cd ..
   61  cd shell_scripting
   62  cp read_print.sh /home/ec2-user/dir1
   63  cd ..
   64  mkdir dir2
   65  cp -r dir2 /home/ec2-user/dir1
   66  cd dir1
   67  ls -lrt
   68  rm -r dir2
   69  cd ..
   70  cd -
   71  cd shell_scripting
   72  cd 
   73  cd shell_scripting
   74  cd ..
   75  cd dir1
   76  mv read_ptint.sh /home/ec2-user/shell_scripting/read.sh
   77  mv read_print.sh /home/ec2-user/shell_scripting/read.sh
   78  cd ..
   79  cd shell_scripting
   80  ls -lrt
   81  mkdir dir1
   82  mv dir1 /home/ec2-user/dir2/dir11
   83  cd ..
   84  cd dir2
   85  ls
   86  rm -rf dir11
   87  cd ../..
   88  cd /home/ec2-user/
   89  pwd
   90  wc read.sh
   91  cd shell_scripting
   92  wc read.sh
   93  wc -l read.sh
   94  wc -c read.sh
   95  wc -w read.sh
   96  cd dir1
   97  ls
   98  deep1
   99  cat deep1
  100  vi deep1
  101  cat deep1
  102  grep "like" deep1
  103  grep -i "Resource " deep1
  104  egrep "Resource " process" "smooth" deep1
  105  egrep "Resource " process" "smooth" deep1
  106  egrep "Resource " " process" "smooth" deep1
  107  grep -e process" -e "smooth" deep1
  108  grep -e "process" -e "smooth" deep1
  109  egrep "process" "smooth" deep1
  110  egrep "process" deep1
  111  egrep -i "Resource" deep1
  112  cd dir1 && deep1
  113  cd dir1
  114  cd ..
  115  cd dir1
  116  cat deep1
  117  cd ..
  118  cd /home/ec2-user/dir1/deep1
  119  cd dir1
  120  ls
  121  cat deep1
  122  cd shell_scripiting
  123  ls
  124  cd shell_scripting
  125  cd ..
  126  cd dir1
  127  sh
  128  cd shell_scripting
  129  cd ..
  130  cd dir11
  131  cd dir1
  132  ls -lrt
  133  cat deep1
  134  grep -l -R "bent" 
  135  grep -w "bent" deep1
  136  grep -w "pr" deep1
  137  grep -w "process" deep1
  138  grep
  139  cd dir1
  140  cat deep1
  141  grep -i "^Resource" deep1
  142  cd dir1
  143  cat deep1
  144  grep "areas$" deep1
  145  grep -v "areas" deep1
  146  grep -v "ideas" deep1
  147  grep "knowledge" deep1
  148  grep -w "knowledge deep1
  149  grep -c "knowledge" deep1
  150  grep -w "knowledge" deep1
  151  grep -l "knowledge" deep1
  152  cd dir1
  153  ls 
  154  cat deep1
  155  cd shell_scripting
  156  ls -lrt
  157  vi age_morethan_30.sh
  158  cd shell_scripting
  159  pwd
  160  ls -lrt
  161  mkdir examples_trail
  162  ls -lrt
  163  cd examples_trail
  164  vi exapmle.sh
  165  sh example.sh
  166  ./example.sh
  167  cat example.sh
  168  ls -lrt
  169  cat exapmle.sh
  170  sh exapmle.sh
  171  ls 
  172  cd dir2
  173  ls 
  174  vi if_else.sh
  175  sh if_else.sh
  176  vi if_else2.sh
  177  sh if_else2.sh
  178  rm is_else.sh
  179  rm if_else.sh
  180  rm if_else2.sh
  181  cd dir1
  182  ls -lt
  183  cd ../dir2
  184  ls -lt
  185  vi ascending_order.sh
  186  sh ascending_order.sh
  187  vi ascending_order.sh
  188  sh ascending_order.sh
  189  vi ascending_order.sh
  190  sh ascending_order.sh
  191  vi ascending_order.sh
  192  sh ascending_order.sh
  193  vi ascending_order.sh
  194  sh ascending_order.sh
  195  vi ascending_order.sh
  196  sh ascending_order.sh
  197  vi ascending_order.sh
  198  sh ascending_order.sh
  199  vi wokr.sh
  200  vi ascending_order.sh
  201  vi wokr.sh
  202  sh wokr.sh
  203  cat ascending_order.sh
  204  cat work.sh
  205  cat wokr.sh
  206  vi ascending_order.sh
  207  sh ascending_order.sh
  208  set -x 
  209  set -x sh asecending_order.sh
  210  vi ascending_order.sh
  211  sh ascending_order.sh
  212  vi wokr.sh
  213  sh wokr.sh
  214  cat wokr.sh
  215  vi wokr.sh
  216  sh wokr.sh
  217  vi wokr.sh
  218  ls-lrt'
  219  cd ..
  220  cat wokr.sh
  221  cd dir2
  222  ls
  223  ls -lrt
  224  vi reverse_string.sh
  225  sh reverse_string.sh
  226  cat reverse_string.sh
  227  vi reverse_string.sh
  228  sh reverse_string.sh
  229  cat reverse_string.sh
  230  vi reverse_string.sh
  231  sh reverse_string.sh
  232  vi reverse_string.sh
  233  sh reverse_string.sh
  234  vi reverse_string.sh
  235  sh reverse_string.sh
  236  vi reverse_string.sh
  237  sh reverse_string.sh
  238  vi reverse_string.sh
  239  rm reverse_string.sh
  240  vi reverse_string.sh
  241  sh reverse_string.sh
  242  vi reverse_string.sh
  243  sh reverse_string.sh
  244  cd dir2
  245  ls -lrt
  246  vi age.sh
  247  sh age.sh
  248  vi age.sh
  249  sh age.sh
  250  vi age.sh
  251  sh age.sh
  252  vi age.sh
  253  sh age.sh
  254  cd dir2
  255  ls -lrt
  256  sh age.sh
  257  vi age.sh
  258  vi add.sh
  259  sh add.sh
  260  vi add.sh
  261  sh add.sh
  262  vi add.sh
  263  sh add.sh
  264  vi add.sh
  265  sh add.sh
  266  vi add.sh
  267  sh add.sh
  268  vi add.sh
  269  sh add.sh
  270  vi add.sh
  271  sh add.sh
  272  vi add.sh
  273  sh add.sh
  274  vi add.sh
  275  ls -lrt
  276  vi if.sh
  277  sh if.sh
  278  vi if.sh
  279  sh if.sh
  280  vi if.sh
  281  sh if.sh
  282  vi if.sh
  283  cd dir2
  284  ls -lrt
  285  cd ../dir1
  286  ls -lrt
  287  cat deep1
  288  grep -l "mgmt" deep1
  289  grep -w "mgmt" deep1
  290  grep -i "^" deep1
  291  grep -i "^Works" deep1
  292  grep "process$" deep1
  293  grep -e "with" -e "like" deep1
  294  egrep "with" -e "like" deep1
  295  egrep "with" "like" deep1
  296  head -2 deep1
  297  head -1 deep1
  298  vi deep1
  299  less deep1
  300   head -5 | tail -1
  301   head -5 | tail -1 deep1
  302  cat deep1
  303  head deep1
  304  tail -2
  305  tail -2 deep1
  306  tail -4
  307  tail -4 deep1
  308  head -6 | tail -1 deep1
  309  head -8 deep1 | tail -1
  310  head -2 deep1 | tail -8
  311  cat deep1
  312  head -8 deep1 | tail -6
  313  vi deep1
  314  head -7 deep1 | tail -2
  315  head -7 deep1 | tail -3
  316  tail -7 deep1 
  317  tail -7 deep1 | head -3
  318  vi ht
  319  cat ht
  320  tail -9 ht | head -1
  321  head -10 ht | tail -1
  322  head -10 ht | tail -9
  323  head -9 ht | tail -9
  324  cat ht
  325  head -5 ht | tail -7
  326  head -5 ht | tail -8
  327  head -5 ht | tail -6
  328  head -5 ht | tail -2
  329  cat ht
  330  tail -4 ht
  331  vi ht
  332  tail -4 ht
  333  tail -6 ht | head -4
  334  head -9 ht | tail -1 | wc -c
  335  head -8 ht | tail -1 | wc -c
  336  cat ht
  337  head -8 ht
  338  tail -1
  339  cd dir2
  340  ls 
  341  cd ../dir1
  342  ls
  343  cat h
  344  cat ht
  345  tail -5 ht | head -2
  346  cat ht
  347  sed 's/linux/gagan/g' ht
  348  cat ht
  349  sed -i 's/linux/gagan/g' ht
  350  cat ht
  351  sed  '2s/unix/deep/g' ht
  352  sed -i '2s/unix/deep/g' ht1
  353  sed -i '2s/unix/deep/g' ht
  354  cat ht
  355  sed -i '2,5s/unix/deep/g' ht
  356  cat ht
  357  cd ../dir2
  358  cat deep1
  359  ls
  360  cd ../dir1
  361  cat deep1
  362  vi deep1
  363  cat deep1
  364  sed '2s/low/gagan/g' deep1
  365  vi deep1
  366  sed '2,5s/month/gagan/g' deep1
  367  sed '$s/your/gagan/g' deep1
  368  sed '2d' deep1
  369  sed '7d' deep1
  370  cat deep1
  371  vi deep1
  372  vi ht
  373  sed '2d' ht
  374  cat ht
  375  sed '2d' ht
  376  cd dir1
  377  la
  378  ls
  379  cat ht
  380  sed '2,5' ht
  381  sed '2,5d' ht
  382  sed '$d' ht
  383  sed -n '3p' ht
  384  sed -n '1,3p' ht
  385  sed -n '2,$p' ht
  386  cd dir2
  387  ls
  388  cd ../dir1
  389  ls -lrt
  390  touch file{1..4}
  391  ls -lrt
  392  ls -lrt | cut -d " " -f 1,2
  393  ls -lrt | cut -d " " -f 1
  394  ls -lrt | cut -d " " -f 1,2,3
  395  ls -lrt | cut -d " " -f 1 | head -1
  396  ls -lrt | cut -d " " -f 1,2,3
  397  ls -lrt | cut -d " " -f 1,2,3 | tail -7
  398  ls -lrt | cut -d " " -f 2
  399  ls -lrt | cut -d " " -f 2 | tail -7
  400  vi exel
  401  cut -d " " -f 1 exel
  402  cut -d "    " -f 1 exel
  403  awk -F " " '{print $1}' exel
  404  awk -F " " '{print $1,2,3,4} exel | head -1
  405  awk -F " " '{print $1,2,3}' | head -
  406  awk -F " " '{print $1,2,3}' exel | head -1
  407  awk -F " " '{print $1,2,3}' exel | tail -4
  408  awk -F " " '{print $1,2,3,4}' exel | tail -4
  409  tail -4 | awk -F " " '{print $1,}' exel
  410  tail -4 exel
  411  tail -5 exel
  412  cat exel
  413  head -1
  414  head -1 exel
  415  ls -lrt
  416  ls -lrt | cut -d " " -f 8
  417  ls -lrt | awk -F " " '{print $8}'
  418  cd dir1
  419  ls -lrt
  420  cat exel
  421  awk -F " " '{print $3}' exel
  422  ls -lrt
  423  ls -lrt | awk -F " " '{print $2}' | tail -8
  424  ls -lrt | awk -F " " '{print $1}' | tail -8
  425  ls -lrt | awk -F " " '{print $(NF-1)}'
  426  ls -lrt | awk -F " " '{print $(NF-3)}'
  427  ls -lrt | awk -F " " '{print $(NF-2)}'
  428  ls -lrt | awk -F " " '{print $(NF)}'
  429  cd dir1
  430  cd dir1
  431  ls -lrt
  432  find -name "deep1"
  433  cd ..
  434  find -name "deep1"
  435  cd dir1
  436  touch Gagan
  437  cd ..
  438  find -iname "Gagan"
  439  find -name "Gagan"
  440  find -type f -mtime -90
  441  find -type f -mtime -10
  442  find -type f -mtime +10
  443  find -type f -mmin +5
  444  find -type f -mmin -5
  445  find -type f -mmin -90
  446  find -type f -mtime -2
  447  find -type f -empty
  448  find -type f -perm 777
  449  find -type f -perm 776
  450  find . -type f -perm 776
  451  find . -type f -perm 0776
  452  find . -type f -empty
  453  cd dir1
  454  ls -lrt
  455  find . -maxdepth3 -type f -empty
  456  find . -maxdepth1
  457  find -maxdepth1
  458  find .-type -f -empty
  459  find . -type ! -empty
  460  find . -type -empty
  461  find . -type f -empty
  462  find . -type f  -empty
  463  find . -type f ! -empty
  464  cd ..
  465  -maxdepth1
  466  find . -maxdepth 1 -type -ht
  467  find . -maxdepth 1 -type -empty
  468  find -maxdepth 1 -type -empty
  469  find -maxdepth 1 -type f -empty
  470  find -maxdepth 2 -type f -empty
  471  find -maxdepth 3 -type f -empty
  472  find -maxdepth 4 -type f -empty
  473  find .-type f -empty |xargs rm 
  474  find . -type f -empty |xargs rm 
  475  su
  476  sudo su
  477  ps -ef
  478  ps -ef | grep apache
  479  kill -9 1818
  480  kill -9 18184
  481  pwd
  482  cd ec2-user
  483  cd dir1
  484  ls
  485  cd ../dir2
  486  ls
  487  vi ascending.sh
  488  sh menu.sh
  489  sh ascending.sh
  490  vi ascending.sh
  491  sh ascending.sh
  492  vi ascending.sh
  493  vi accendingorder.sh
  494  sh accendingorder.sh
  495  vi accendingorder.sh
  496  sh accendingorder.sh
  497  vi accendingorder.sh
  498  vi decendingorder.sh
  499  sh decendingorder.sh
  500  vi decendingorder.sh
  501  sh decendingorder.sh
  502  vi decendingorder.sh
  503  ls -lrt
  504  vi menu.sh
  505  vi reverse_string.sh
  506  su -l
  507  ls -lrt
  508  umask 000
  509  ls -lrt
  510  cd dir1
  511  ls
  512  touch gagna
  513  find . -type f -empty | xargs rm
  514  ls 
  515  ps
  516  cd dir2
  517  ls
  518  vi reverse_string.sh
  519  sh reverse_string.sh
  520  vi reverse_string.sh
  521  sh reverse_string.sh
  522  vi reverse_string.sh
  523  cd dir1
  524  ls
  525  cd ../dir2
  526  ls
  527  ci assending.sh
  528  vi ascending.sh
  529  sh ascending.sh
  530  vi ascending.sh
  531  sh ascending.sh
  532  vi ascending.sh
  533  sh ascending.sh
  534  vi ascending.sh
  535  sh ascending.sh
  536  vi ascending.sh
  537  sh ascending.sh
  538  vi ascending.sh
  539  sh ascending.sh
  540  vi ascending.sh
  541  vi menu.sh
  542  sh menu.sh
  543  vi menu.sh
  544  sh menu.sh
  545  vi menu.sh
  546  sh menu.sh
  547  vi menu.sh
  548  sh menu.sh
  549  vi menu.sh
  550  sh menu.sh
  551  vi menu.sh
  552  pwd
  553  cd dir2
  554  ls
  555  cd /dir2
  556  cd /dir1
  557  cd ../dir1
  558  ls
  559  vi uniq_sort
  560  uniq uniq_sort
  561  vi uniq_sort
  562  sort unit_sort
  563  sort uniq_sort
  564  vi uniq_sort
  565  sort uniq_sort
  566  uniq uniq_sort
  567  $$
  568  $
  569  cd dir1
  570  pwd
  571  ln -s /home/ec2-user/dir1 softlink
  572  ls -lrt
  573  cd softlink
  574  ls
  575  cd ..
  576  rm softlink
  577  ln /home/ec2-user/dir1 hardlink
  578  cd ..
  579  ln /home/ec2-user/dir1 hardlink
  580  cd dir2
  581  pwd
  582  cd ..
  583  ln /home/ec2-user/dir2 hlink
  584  ln /home/ec2-user/dir2/deep1 hlink
  585  ln /home/ec2-user/dir2/deep1 hlink1
  586  ln /home/ec2-user/dir2 hlink
  587  ln dir2 hlink
  588  cd dir1
  589  ln /home/ec2-user/dir1 hardlink
  590  man du
  591  cd dir1
  592  ls
  593  vi practice
  594  vi practice.sh
  595  sh practice.sh
  596  vi practice.sh
  597  cd dir1
  598  ls
  599  vi practice.sh
  600  sh practice.sh
  601  vi practice.sh
  602  sh practice.sh
  603  wq!
  604  vi practice.sh
  605  sh practice.sh
  606  vi practice.sh
  607  sh practice.sh
  608  vi practice.sh
  609  sh practice.sh
  610  vi practice.sh
  611  sh practice.sh
  612  vi practice.sh
  613  sh practice.sh
  614  vi practice.sh
  615  sh practice.sh
  616  sh practice.sh
  617  vi practice.sh
  618  sh practice.sh
  619  vi practice.sh
  620  sh practice.sh
  621  echo $@
  622  echo $!
  623  echo $*
  624  cd dir1
  625  ls
  626  vi practice
  627  vi practice.sh'
  628  vi practice.sh
  629  sh practice.sh
  630  vi practice.sh
  631  sh practice.sh
  632  cd dir1
  633  ls'
  634  ls
  635  vi practice.sh
  636  sh practice.sh
  637  vi practice.sh
  638  sh practice.sh
  639  vi practice.sh
  640  set -x sh practice.sh
  641  sh practice.sh
  642  8
  643  vi practice.sh
  644  sh practice.sh
  645  cd dir1
  646  ls
  647  vi practice.sh
  648  sh practice.sh
  649  vi practice.sh
  650  sh practice.sh
  651  vi practice.sh
  652  sh practice.sh
  653  vi practice.sh
  654  sh practice.sh
  655  sh practice.sh.
  656  sh practice.sh
  657  2 5 6
  658  sh practice.sh
  659  vi practice.sh
  660  sh practice.sh
  661  vi practice.sh
  662  sh practice.sh
  663  cd dir1
  664  vi practice.sh
  665  sh practice.sh
  666  find -name practice.sh
  667  find .-mtime-5
  668  find .-mtime -5
  669  find .-mtime +5
  670  find .-mmin +6
  671  vi while.sh
  672  sh while.sh
  673  find -type f -empty
  674  find . -type f -empty
  675  find .-mtime +5
  676  cd dir1
  677  ls
  678  vi tar.sh
  679  sh tar.sh
  680  vi tar.sh
  681  sh tar.sh
  682  vi tar.sh
  683  sh tar.sh
  684  vi tar.sh
  685  sh tar.sh
  686  vi tar.sh
  687  sh tar.sh
  688  vi tar.sh
  689  sh tar.sh
  690  vi tar.sh
  691  sh tar.sh
  692  crantab -l
  693  ls
  694  vi tar.sh
  695  clear
  696  cd dir1
  697   touch file1 file2
  698  ls
  699  vi file2
  700  cat file1
  701  ls
  702  cat file2
  703  cp file2 file1
  704  cat file1
  705  cat file2
  706  mv file2 file1
  707  cat file2
  708  cat file1
  709  sudo su
  710  sudo useradd
  711  sudo useradd gagan
  712  sudo passwd gagan
  713  sudo su
  714  chown gagan
  715  sudo chown raj gagan
  716  sudo chown gagan
  717  su gagan
  718  ls
  719  pwd
  720  crontab -e
  721  crontab -l
  722  ps -ef
  723  ping www.utilitysolutions.in
  724  ssh
  725  ls -lrt
  726  grep "mega" file1
  727  grep "mega" | grep -v
  728  grep "mega" file1 
  729  grep "mega" file1 | grep -v
  730  grep "mega" file1
  731  grep -v file1
  732  grep -v "mega" file1
  733  ls -lrt
  734  crantab -e
  735  crontab -e
  736  crontab -l
  737  vi if.sh
  738  sh if.sh
  739  vi while.sh
  740  sh while.sh
  741  vi while.sh
  742  sh while.sh
  743  vi while.sh
  744  sh while.sh
  745  vi while.s
  746  vi while.sh
  747  sh while.sh
  748  vi while.sh
  749  sh while.sh
  750  vi while.sh
  751  sh while.sh
  752  vi while.sh
  753  sh while.sh
  754  vi while.sh
  755  sh while.sh
  756  vi while.sh
  757  sh while.sh
  758  vi while.sh
  759  sh while.sh
  760  vi while.sh
  761  sh while.sh
  762  vi while.sh
  763  sh while.sh
  764  vi while.sh
  765  sh while.sh
  766  vi while.sh
  767  sh while.sh
  768  vi while.sh
  769  sh while.sh
  770  vi while.sh
  771  sh while.sh
  772  vi while.sh
  773  sh while.sh
  774  vi while.sh
  775  ls
  776  vi if.sh
  777  vi while.sh
  778  sh while.sh
  779  vi try.sh
  780  sh try.sh
  781  1 2
  782  sh try.sh
  783  vi try.sh
  784  sh try.sh
  785  vi try.sh
  786  vi while.sh
  787  vi 4tab.sh
  788  sh 4tab.sh
  789  vi elif.sh
  790  sh elif.sh
  791  vi elif.sh
  792  sh elif.sh
  793  vi elif.sh
  794  vi try.sh
  795  vi elif.sh
  796  sh elif.sh
  797  sh try.sh
  798  clear
  799  vi elif.sh
  800  sh elif.sh
  801  vi elif.sh
  802  vi for.sh
  803  cd dir1
  804  clear
  805  sudo yum install git -y
  806  history
  807  where is git
  808  which git
  809  which appache
  810  git --v
  811  git --version
  812  cd shell_scripting
  813  ls
  814  ls -lrt
  815  vi read_print.sh
  816  cd ..
  817  mkdir first repository
  818  cd first repository
  819  clear
  820  git init
  821  ls -lart
  822  cd .git\
  823  cd .git
  824  ls -lrt
  825  cat config
  826  cd ..
  827  clear
  828  git config --global init.defaultBranch master
  829  git branch
  830  git config --global user.name "gagan"
  831  git config --global user.email "gaganm.k78@gmail.com"
  832  cd .git
  833  ls -lrt
  834  cat config
  835  clear
  836  cd first
  837  ls -lrt
  838  git status
  839  git branch -M master
  840  git add print_char.sh
  841  git status
  842  cd ../shell_scripting
  843  ls
  844  cp add_number.sh /home/ec2-user/first
  845  cd ../first
  846  ls
  847  git status
  848  git commit -m "adding a print the character of the script"
  849  git status
  850  git remote -v
  851  git push
  852  cd first repository
  853  git remote add origin https://github.com/GagandeepMK/first-repository-.git
  854  cd first
  855  gi status
  856  git status
  857  git push
  858  git push --set-upstream origin master
  859  clear
  860  cd first repository
  861  cp ../shell_scripting/*
  862  cd ../shell_scripting
  863  ls
  864  ls -lrt
  865  rm -rf exaples_traill
  866  cp ../shell_scripting/*
  867  cp ../shell_scripting/* .
  868  ls -lrt
  869  cd ../first repository
  870  ls -lrt
  871  cp ../shell_scripting/* 
  872  cd ../shell_scripting
  873  cd ../first repository
  874  cp ../shell_scripting/leap_year.sh
  875  cp ../shell_scripting/print_char.sh
  876  pwd
  877  cp ../shell_scripting/print_char.sh /home/ec2-user/first
  878  ls -lrt
  879  git status
  880  cd .git
  881  cd first repository
  882  ls
  883  git status
  884  cd first
  885  cd first
  886  ls
  887  git status
  888  git push
  889  git push --set-upstream origin master
  890  cd first
  891  git status
  892  git add add_number.sh
  893  git commit -m "adding the numbers script"
  894  git push
  895  uname -v
  896  uname -version
  897  uname --version
  898  uname
  899  uname -a
  900  uname
  901  uname -v
  902  uname -version
  903  uname -a
  904  uname --v
  905  uname -v
  906  uname -a
  907  cd ~/.ssh/
  908  ls
  909  ls
  910  rm -rf repository
  911  ls
  912  cd first
  913  ls
  914  git status
  915  cd ../shell_scripting
  916  ls
  917  cp if_else.sh leap_year.sh /home/ec2-user/first
  918  cd ../shell_scripting
  919  cd ../first
  920  ls
  921  git status
  922  git add if_else.sh
  923  git commit -m "script on if else statement"
  924  git status
  925  git push
  926  git status
  927  ~/.ssh/
  928  ~/ .ssh/
  929  .ssh/
  930  ~/.ssh/
  931  cd ..
  932  ~/.ssh/
  933  cd ~/.ssh/
  934  ls
  935  ls -lrt
  936  cd first
  937  cd /home/ec2-user/
  938  ls
  939  cd dir1
  940  ls
  941  vi every50_lin.sh
  942  ls
  943  cd dir1
  944  ls
  945  vi gg.sh
  946  cat gg.sh
  947  sed '0~2 a\\' gg.sh
  948  sed '0~2 abcd a\\' gg.sh
  949  rm gg.sh
  950  sed '0~2 abcd a\\' gg
  951  sed '0~2 abcd a\' gg
  952  sed '0~2 abcd \' gg
  953  sed '0~3 abcd \' gg
  954  sed '0~4 HEART \' gg
  955  sed -e '0~4 HEART \' gg
  956  sed -e '0~4 heart \' gg
  957  sed -e '0~3 heart \' gg
  958  sed -e '0~3 heart\' gg
  959  sed -e '0~3 hear\' gg
  960  sed '0~3 hear \' gg
  961  sed '0~2 abcd \' gg
  962  sed '0~4 abcd \' gg
  963  sed '0~4 ABCD \' gg
  964  sed -i '0~4 ABCD \' gg
  965  sed -i '0~4 aBC\' gg
  966  sed '0~4 aBC\' gg
  967  sed '0~4 aABCD\' gg
  968  cat gg
  969  rm gg
  970  vi gg
  971  sed '0~4 aABCD\' gg
  972  up time
  973  ps
  974  date
  975  uptime
  976  cd dir1
  977  ls
  978  sudo chattr +i -v practice.sh
  979  ls
  980  rm practice.sh
  981  ls
  982  sudo chattr +i -v gg
  983  sudo chattr +i gg
  984  rm gg
  985  cd ..
  986  mkdir dir7
  987  sudo chattr +i V dir7
  988  ls
  989  sudo chattr +i -RV dir7
  990  rm -rf dir7
  991  history >>history.sh
